### PR TITLE
add a crane-server

### PR DESCRIPTION
**Pull request by kitianFresh**

### CODE REVIEW

**Comment by mfanjie** [comment]
is `crane-gateway` or `dashboard-gateway` a better name than `crane-apiserver`?

**Comment by kitianFresh** [comment]
> is `crane-gateway` or `dashboard-gateway` a better name than `crane-apiserver`?

changed it to crane-server after discussed with qmhu

File: cmd/crane-server/app/options/grafana_options.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
+package options
+
+import (
+	"fmt"
+	"os"
+
+	"gopkg.in/gcfg.v1"
+
+	"github.com/spf13/pflag"
```
Comment (mfanjie) [inline cmd/crane-server/app/options/grafana_options.go:9]: above two imports can be combined to same section.

File: cmd/crane-server/app/server.go [hunk 2]
```diff
@@ -0,0 +1,54 @@
+package app
+
+import (
+	"context"
+	"flag"
+
+	"github.com/spf13/pflag"
+
+	"github.com/spf13/cobra"
```
Comment (mfanjie) [inline cmd/crane-server/app/server.go:9]: import

File: examples/grafana-config.ini [hunk 3]
```diff
@@ -0,0 +1,7 @@
+[service]
```
Comment (mfanjie) [inline examples/grafana-config.ini:1]: `grafana-config.ini` to `grafana.conf`?

File: pkg/server/service/dashboard/dashboard.go [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+	PanelEmbeddings(ctx context.Context, useRange bool, from time.Time, to time.Time) (*PanelList, error)
+}
+
+type manager struct {
+	config        *GrafanaConfig
+	grafanaClient *grafanasdk.Client
+}
+
+func NewManager(config *GrafanaConfig) (*manager, error) {
+	//var gc *grafanaclient.Client
```
Comment (mfanjie) [inline pkg/server/service/dashboard/dashboard.go:48]: remove unused code

**Pull request closed without merging**